shared/fs/footer/upload.desktop.js

```diff
@@ -85,6 +85,7 @@ const stylesBox = {
   alignItems: 'center',
   justifyContent: 'center',
   maxHeight: height,
+  height,
```

Comment:
doesn't this make `maxHeight` redundant?

Reply:
The original change from Brian was to change from `height` to `maxHeight`, so I'm guessing it `maxHeight` was needed for forcing it into a single line and show the ellipsis and kept both.